Content.Client/IconSmoothing/IconSmoothSystem.cs

```diff
@@ -42,6 +46,10 @@ public override void Initialize()
 
         private void OnStartup(EntityUid uid, IconSmoothComponent component, ComponentStartup args)
         {
+            //Randomize visual
+            if (component.RandomStates.Count > 0)
+                component.StateBase = _random.Pick(component.RandomStates);
```

Comment:
This should check if StateBase has already been set (so loading initialized maps preserves the existing state)

Ideally, this would be randomized at mapinit instead as well, but that seems to be a taller ask.

Reply:
StateBase always set. its have dafault "base" value

Reply:
also its client-side component and system, server dont save this data

Reply:
The randomization should probably be shared, then (Separate component?). Otherwise clients will get inconsistent visuals.

Reply:
The main reason why I don't want to add this to the shared is that storing random information from RandomSprite on the server loads all the objects on procedurally generated biomes, and clogs up memory.